PROTO_tests/tests/stats_test.py

```diff
@@ -0,0 +1,168 @@
+import numpy as np
+import pandas as pd
+import pytest
+
+import arkouda as ak
+
+
+class TestStats:
+    @classmethod
+    def setup_class(cls):
+        cls.x = ak.arange(10, 20)
+        cls.npx = np.arange(10, 20)
+        cls.pdx = pd.Series(cls.npx)
+        cls.y = ak.array([2, 1, 4, 5, 8, 12, 18, 25, 96, 48])
+        cls.npy = np.array([2, 1, 4, 5, 8, 12, 18, 25, 96, 48])
+        cls.pdy = pd.Series(cls.npy)
+        cls.u = ak.cast(cls.y, ak.uint64)
+        cls.npu = np.array([2, 1, 4, 5, 8, 12, 18, 25, 96, 48], dtype=np.uint64)
+        cls.pdu = pd.Series(cls.npu)
+        cls.b = ak.arange(10) % 2 == 0
+        cls.npb = np.arange(10) % 2 == 0
+        cls.pdb = pd.Series(cls.npb)
+        cls.f = ak.array([i**2 for i in range(10)], dtype=ak.float64)
+        cls.npf = np.array([i**2 for i in range(10)], dtype=np.float64)
+        cls.pdf = pd.Series(cls.npf)
+        cls.s = ak.array([f"string {i}" for i in range(10)])
+        cls.c = ak.Categorical(cls.s)
+        cls.arks = [cls.x, cls.y, cls.u, cls.b, cls.f]
+        cls.nmps = [cls.npx, cls.npy, cls.npu, cls.npb, cls.npf]
+        cls.pands = [cls.pdx, cls.pdy, cls.pdu, cls.pdb, cls.pdf]
+
+    def test_mean_var_and_std(self):
+        for ark, npy in zip(self.arks, self.nmps):
+            assert ark.var() == pytest.approx(npy.var())
+            assert ark.std() == pytest.approx(npy.std())
+
+        for ark, pand in zip(self.arks, self.pands):
+            assert ark.mean() == pytest.approx(pand.mean())
+            assert ark.var(ddof=1) == pytest.approx(pand.var())
+            assert ark.std(ddof=1) == pytest.approx(pand.std())
+
+    def test_cov_and_corr(self):
+        # test that cov and corr variations are equivalent
+        for fn in "cov", "corr":
+            for arg in [
+                getattr(self.pdx, fn)(self.pdy),
+                getattr(self.y, fn)(self.x),
+                getattr(ak, fn)(self.x, self.y),
+                getattr(ak, fn)(self.y, self.x),
+            ]:
+                assert getattr(self.x, fn)(self.y) == pytest.approx(arg)
+
+            # test int, bool, and float with other types
+            for ark, pand in [
+                (self.u, self.pdu),
+                (self.b, self.pdb),
+                (self.f, self.pdf),
+            ]:
+                assert getattr(self.x, fn)(ark) == pytest.approx(getattr(self.pdx, fn)(pand))
+                assert getattr(self.b, fn)(ark) == pytest.approx(getattr(self.pdb, fn)(pand))
+                assert getattr(self.f, fn)(ark) == pytest.approx(getattr(self.pdf, fn)(pand))
+
+            # test uint with self (other cases covered above)
+            assert getattr(self.u, fn)(self.u) == pytest.approx(getattr(self.pdu, fn)(self.pdu))
```

Comment:
Now that we have this nice condensed format, I don't see a reason to special case `u`
```suggestion
                assert getattr(self.f, fn)(ark) == pytest.approx(getattr(self.pdf, fn)(pand))
                assert getattr(self.u, fn)(ark) == pytest.approx(getattr(self.pdu, fn)(pand))
```